.devops/deploy-pipelines.yml

```diff
@@ -172,7 +174,8 @@ stages:
             parameters:
               env: 'PROD'
               version: '$(Build.SourceBranchName):$(Build.SourceVersion)'
-              api_url: 'https://api-rtp.cstar.pagopa.it/rtp/v1'
+              vite_api_url: 'https://api-rtp.dev.cstar.pagopa.it/rtp'
```

Comment:
same observation: param name must match the one in the setup env. `vite_api_url` here and `api_url` there